tests/test_lorax.py

```diff
@@ -1,4 +1,9 @@
 """Tests for Lorax."""
+# TODO: Figure out how to do this optimally
+import os
+import sys
+project_path = os.path.join(os.path.dirname(__file__), '../')
+sys.path.append(project_path)
```

Comment:
Pretty sure you shouldn't have to do this... where are you invoking the tests from (e.g. this directory or its parent)? At least according to this stackoverflow: 

https://stackoverflow.com/questions/1896918/running-unittest-with-typical-test-directory-structure